source/crud/query/specify-documents-to-return.txt

```diff
@@ -270,50 +239,31 @@ The following example returns documents with the ``length`` value of
 ``"1104"``. The results are sorted in alphabetical order, skipping the first
 document and includes only the first result:
 
-.. code-block:: javascript
-
-   // define a query to look for length value of 1104
-   const query = {length: "1104"};
-   const options = {
-      // sort in alphabetical (1) order by title
-      sort : { title: 1 },
-      // omit the first document
-      skip : 1,
-      // returns only the first result
-      limit: 1,
-   }
-   const cursor = myColl.find(query, options);
-   for await (const doc of cursor) {
-     console.dir(doc);
-   }
-
-.. code-block:: json
-   :copyable: false
-
-   { "_id": 4, "title": "Infinite Jest", "author": "Wallace", "length": 1104 }
-
-.. note::
-
-   The order in which you call these methods doesn't change the documents
-   that are returned. The driver automatically reorders the calls to perform the
-   sort and skip operations first, and the limit operation afterward.
-
-You can also limit, sort, and skip results by chaining each method to the ``find`` method.
-The following example specifies the same query as the preceding example:
-
 .. io-code-block::
 
    .. input:: 
       :language: javascript
 
-      myColl.find(query).sort({ title: 1 }).skip(1).limit(1);
+      const query = {length: "1104"};
+
+      const cursor = myColl.find(query).sort({ title: 1 }).skip(1).limit(1);
+
+      for await (const doc of cursor) {
+         console.dir(doc);
+      }
 
    .. output::
       :language: json
       :visible: false
       
       { "_id": 4, "title": "Infinite Jest", "author": "Wallace", "length": 1104 }
 
+.. note::
+
+   The order in which you call these methods doesn't change the documents
+   that are returned. The driver automatically reorders the calls to perform the
+   sort and skip operations first, and the limit operation afterward.
```

Comment:
```suggestion
   that are returned.
```

This isn't accurate - each chaining method just corresponds to an option that we send to the server.  ex: `.limit()` sets the `limit` option